Holodeck/Environments.py

```diff
@@ -94,6 +96,12 @@ def __init__(self, agent_definitions, binary_path=None, task_key=None, height=51
         # Subscribe settings
         self._reset_ptr = self._client.subscribe_setting("RESET", [1], np.bool)
         self._reset_ptr[0] = False
+        self._command_bool_ptr = self._client.subscribe_setting("command_bool", [1], np.bool)
+        self._command_buffer_ptr = self._client.subscribe_setting("command_buffer", [int(1048576 / 8)], np.byte)
```

Comment:
Could you please explain magic numbers? It would also be better to just give the result of the calculation since it's always the same.